nodebug.go

```diff
@@ -2,14 +2,13 @@
 
 package ipfscluster
 
-// This is our default logs levels
+// These are our default log levels
```

Comment:
A general comment about nodebug, silent, debug, and logging: is it possible to put them somewhere a little more cohesive than simply in the main ipfscluster directory?  Until right now I hadn't put it together that all these files were part of the logging system and think it would aid people understanding the codebase to somehow group them better.  I can believe right now that they need to be part of ipfscluster and each use `init` functions so as it stands they can't be put into a single file or put into a subfolder.  Maybe it doesn't need to happen in this PR, but if you agree then maybe we could open an issue for this?

Reply:
Actually, I have realized that it is possible to pass flags to tests just like any executable. This nodebug etc files were just a hack to get such behaviour from long ago. I'll complete the PR getting rid of them.

Reply:
ok, how is looking now?